Soomsil-USaint/Application/Feature/Home/View/HomeView.swift

```diff
@@ -1,335 +1,187 @@
-                )
-            }
-            .foregroundColor(Color(red: 0.06, green: 0.07, blue: 0.07))
-            .padding(.horizontal, Dimension.MainSpacing.horizontal)
-        })
-    }
+    // MARK: - GradeSection
+    struct GradeInfo: View {
+        var reportCard: TotalReportCard
 
-    private func detailGradeListView(average: Float, credit: Float, graduateCredit: Float) -> some View {
-        VStack(spacing: Dimension.DetailSpacing.vertical) {
-            HStack {
-                Text("평균학점").font(YDSFont.body1)
-                Spacer()
-                Text(String(format: "%.2f", average)).font(YDSFont.subtitle2)
-                    .foregroundColor(Color(red: 0.51, green: 0.43, blue: 0.93))
-                Text("/ \(String(format: "%.2f", 4.50))").font(YDSFont.subtitle3)
-                    .foregroundColor(Color(red: 0.56, green: 0.58, blue: 0.6))
-                    .padding(.leading, -4)
-            }
-            .frame(height: 23)
-            .padding(.vertical, Dimension.DetailPadding.vertical)
-            .padding(.horizontal, Dimension.DetailPadding.horizontal)
-
-            Divider()
-                .padding(.horizontal, 13)
-
-            HStack {
-                Text("취득학점").font(YDSFont.body1)
-                Spacer()
-                Text(String(format: "%.1f", credit)).font(YDSFont.subtitle2)
-                    .foregroundColor(Color(red: 0.51, green: 0.43, blue: 0.93))
-                Text("/ \(String(Int(graduateCredit)))").font(YDSFont.subtitle3)
-                    .foregroundColor(Color(red: 0.56, green: 0.58, blue: 0.6))
-                    .padding(.leading, -4)
-            }
-            .frame(height: 23)
-            .padding(.vertical, Dimension.DetailPadding.vertical)
-            .padding(.horizontal, Dimension.DetailPadding.horizontal)
+        let onSemesterListPressed: () -> Void
 
-            // MARK: - FIX
+        var body: some View {
             Button(action: {
-                path.append(StackView(type: .SemesterList))
+                onSemesterListPressed()
             }, label: {
-                Text("전체 학기 성적 보기")
-                    .font(Font.custom("Apple SD Gothic Neo", size: 15))
-                    .foregroundColor(isLatestSemesterNotYetConfirmed ? .white : Color(red: 0.15, green: 0.15, blue: 0.16))
-                    .frame(height: 39, alignment: .center)
-                    .frame(maxWidth: .infinity)
-                    .background(isLatestSemesterNotYetConfirmed ? Color(red: 0.51, green: 0.43, blue: 0.93) : Color(red: 0.95, green: 0.96, blue: 0.97))
-                    .cornerRadius(4)
+                VStack {
+                    VStack(spacing: 0) {
+                        HStack {
+                            Text("내 성적")
+                                .font(YDSFont.title3)
+                            Spacer()
+                        }
+                        .padding(.leading, 4)
+                        .padding(.top, 20)
```

Comment:
```suggestion
                    VStack(alignment: .leading, spacing: 0) {
                        Text("내 성적")
                            .font(YDSFont.title3)
                            .padding(.leading, 4)
                            .padding(.top, 20)
```

HStack을 사용하지 않고, 상위 VStack의 alignment로 위치를 조정하는 방법도 있어요!
아래 화면들이 모두 전체를 차지해서 이 방법도 가능합니다.